### ISSUE TITLE
Running stressberry on RPi4 in a solid KKSB milled case

### ISSUE THREAD

**Issue by JohBod** [nschloe/stressberry#31]
My new KKSB case soaked up the CPU heat pretty well and the large case area forwarded the heat into the air through convection. No fan in sight, it appears to stay near 60 degrees C, no clock throttling detected.

![rpi4stress](https://user-images.githubusercontent.com/27567970/63807224-9acef600-c91d-11e9-8c8f-92953c59abe7.jpg)

![out](https://user-images.githubusercontent.com/27567970/63807294-b20de380-c91d-11e9-9692-5effd308fdd4.png)

**Comment by nschloe** [comment]
What a nice case! Could you also attach the data file so I can merge it into the figure on the README?

**Comment by JohBod** [comment]
I replied to Github's email notification about your message above about a week ago. I got no response so, in case you didn't get it, i'll attach the file here as well. The old graph above is from running the test with the case partially covered with a paper. The attached file is from running the test with the case standing free and alone on the table to allow free air circulation.

[out.txt](https://github.com/nschloe/stressberry/files/3596822/out.txt)
